app/uk/gov/hmrc/pillar2/controllers/RegistrationCacheController.scala

```diff
@@ -46,12 +46,10 @@ class RegistrationCacheController @Inject() (
   }
 
   def get(id: String): Action[AnyContent] = authenticate.async { implicit request =>
-    implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequestAndSession(request, request.session)
-
-    logger.debug(s"[Session ID: ${SessionIdHelper.sessionId(hc)}] - controllers.RegistrationCacheController.get: Authorised Request " + id)
+    logger.debug(s"controllers.RegistrationCacheController.get: Authorised Request " + id)
     repository.get(id).map { response =>
       logger.debug(
-        s"[Session ID: ${SessionIdHelper.sessionId(hc)}] - controllers.RegistrationCacheController.get: Response for request Id $id is $response"
+        s"controllers.RegistrationCacheController.get: Response for request Id $id is $response"
       )
       response.map(Ok(_)).getOrElse(NotFound)
     }
```

Comment:
Need a unit test remove method when response is false.

Reply:
Done